sources/Valkey.Glide/Abstract/ValkeyServer.cs

```diff
@@ -101,4 +100,84 @@ public async Task<long> ClientIdAsync(CommandFlags flags = CommandFlags.None)
         Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
         return await _conn.Command(Request.ClientId(), MakeRoute());
     }
+
```

Comment:
nit: Please remove the extra lines between methods. Typically we have one blank line between methods. Code should be consistently formatted.